scripts/src/main/resources/scripts/command/mvn

```diff
@@ -30,6 +30,28 @@ function doSetup() {
         echo "Successfully created ${M2_DIR}/settings-security.xml"
         MASTER_PASSWORD=
       fi
+      if [ ! -e "${M2_DIR}/settings.xml" ]
+      then
+        echo "Starting to create ${M2_DIR}/settings.xml"
+        local secret
+        local crypted
+        local settings
+        settings=$(cat "${SETTINGS_PATH}/devon/conf/.m2/settings.xml")
+        if ! doIsBatch
+        then
+          # shellcheck disable=SC2013
+          for var in $( grep -o '\$\[.*\]' "${SETTINGS_PATH}/devon/conf/.m2/settings.xml" | sort -u)
+          do
+            read -r -p "Please enter secret value for variable ${var}: " secret
```

Comment:
Does this allow empty values? What if you don't have passwords at the moment? E.g. A person did not setup sonatype login yet?

Reply:
sure. You can type any password, login, or nonsense here including the empty string. I tested with empty string and `mvn -ep` will give you `{}` that then ends up as the replacement. I think this is OK.